src/style/aframe.css

```diff
@@ -54,7 +54,7 @@
   right: 0 !important;
   bottom: 0 !important;
   z-index: 999999 !important;
-  position: fixed;
+  position: fixed !important;
```

Comment:
Added a comment. If developers want to override, they can do `<a-scene embedded>` which removes these styles and allows styling of the canvas. That's also already documented.